packages/server/lib/open_project.js

```diff
@@ -84,101 +84,100 @@ const moduleFactory = () => {
 
       debug('open project url %s', url)
 
-      return openProject.getConfig()
-      .then((cfg) => {
-        _.defaults(options, {
-          browsers: cfg.browsers,
-          userAgent: cfg.userAgent,
-          proxyUrl: cfg.proxyUrl,
-          proxyServer: cfg.proxyServer,
-          socketIoRoute: cfg.socketIoRoute,
-          chromeWebSecurity: cfg.chromeWebSecurity,
-          isTextTerminal: cfg.isTextTerminal,
-          downloadsFolder: cfg.downloadsFolder,
-        })
+      const cfg = openProject.getConfig()
 
-        // if we don't have the isHeaded property
-        // then we're in interactive mode and we
-        // can assume its a headed browser
-        // TODO: we should clean this up
-        if (!_.has(browser, 'isHeaded')) {
-          browser.isHeaded = true
-          browser.isHeadless = false
-        }
+      _.defaults(options, {
```

Comment:
this entire chunk is the same, removing the promise removed `then` and a ton of white space which makes the code look different